scripts/index_services.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+"""Read services from the API endpoint and write to search-api for indexing.
+
+Usage:
+    index_services.py <search_endpoint> <search_access_token> <api_endpoint>
+              <api_access_token> [options]
+
+    --serial        Do not run in parallel (useful for debugging)
+
+Example:
+    ./index_services.py http://search-api myToken http://data-api token
+"""
+
+from __future__ import print_function
+from six.moves import map
+import sys
+import multiprocessing
+from datetime import datetime
+
+from docopt import docopt
+from dmutils import apiclient
+
+
+def request_services(api_url, api_access_token, page=1):
+
+    data_client = apiclient.DataAPIClient(
+        api_url,
+        api_access_token
+    )
+
+    while page:
+        try:
+            services_page = data_client.find_services(page=page)
+        except apiclient.APIError as e:
+            print('API request failed: {}'.format(e.message), file=sys.stderr)
+            return
+
+        for service in services_page['services']:
+            yield service
+
+        if list(filter(lambda l: l['rel'] == 'next', services_page['links'])):
```

Comment:
This will break when [this story](https://www.pivotaltracker.com/story/show/94576802) is done. I've added a link to this pull request to that story.